src/reports/components/report-sections/tab-stops-report-instance-list.scss

```diff
@@ -4,9 +4,8 @@
 @import '../../../common/styles/fonts.scss';
 
 .instance-list {
-    padding-left: 20px;
-
     th {
         color: $primary-text;
+        padding: 14px 0px 14px 20px;
```

Comment:
Could you instead write out `padding-left`/`padding-right`/etc? The more verbose form is a little easier for a maintainer to understand without looking up the order (we've had a few bugs slip by code review in the past because of the shorthand forms).

Probably the ideal would be for us to enforce this with a lint rule, humans leaving style comments is terrible. Stylelint has a [property-disallowed-list](https://stylelint.io/user-guide/rules/list/property-disallowed-list) rule we could use for this, maybe we should add that to our lint collection...